### ISSUE TITLE
Recurrence problems

### ISSUE THREAD

**Issue by xyz-hcy** [IS2AI/trimodal_person_verification#1]
Dear authors:
I use the same data and model to reproduce the following commands:

> python trainSpeakerNet.py --model ResNetSE34Multi --modality rgb --log_input True --trainfunc angleproto --max_epoch 600 --batch_size 100 --nPerSpeaker 9 --max_frames 200 --eval_frames 200 --weight_decay 0.01 --seed 1 --save_path exps/rgb/exp1

The model training results are shown in the figure below. Loss decreases, TEER/TACC increases.The VEER of the validation set is stable at 27. What is the reason for the result？
![image](https://user-images.githubusercontent.com/55769667/173014033-db3f056a-0923-4c50-af6d-fb327ea540fb.png)
Thanks in advance.

Regards

**Comment by songhaozhen** [comment]
hello,could i add you VX or QQ,i think i lost the trained model and train list,thank you very much
QQ2250050388
